packages/ui/src/components/Radio/mixins.less

```diff
@@ -0,0 +1,7 @@
+.condo-radio-gradient-border(@inner-color, @border-color) {
+  border: double 1px transparent;
```

Comment:
Why double?

Reply:
And we also have default width in tokens

Reply:
> Why double?

That's the way to deal with gradient border color